docs/lang/articles/basic/sparse.md

```diff
@@ -170,31 +172,52 @@ The bitmasked SNodes are like dense SNodes with auxiliary activity values.
 
 ### Dynamic SNode
 
-To support variable-length fields, Taichi provides dynamic SNodes. The code snippet below first creates a 5x1 dense block (line 2). Then each cell of the dense block contains a variable-length dynamic container (line 3). The maximum length of the dynamic container is 5. In the `make_lists()` function, you can use `ti.append()` to add a value to the end of a dynamic SNode. `x.parent()` is the same as `pixel`. The dense field `l` stores the length of each dynamic SNode.
+To support variable-length fields, Taichi provides dynamic SNodes.
 
-```python {3} title=dynamic.py
-x = ti.field(ti.i32)
-block = ti.root.dense(ti.i, 5)
-pixel = block.dynamic(ti.j, 5)
-pixel.place(x)
-l = ti.field(ti.i32)
-ti.root.dense(ti.i, 5).place(l)
-
-@ti.kernel
-def make_lists():
-    for i in range(5):
-        for j in range(i):
-            ti.append(x.parent(), i, j * j)  # ti.append(pixel, i, j * j)
-        l[i] = ti.length(x.parent(), i)  # [0, 1, 2, 3, 4]
-```
+The first argument of `dynamic` is the axis, the second argument is the maximum length,
```

Comment:
```suggestion
The first argument of `dynamic` is the axis, the second the maximum length,
```

Reply:
Will simply writing `the second the maximum length` confuse the readers? (For example I am a little confused) How about `the second one is the maximum length`?

Reply:
We can use the original version then :)